DrupalExtended/ruleset.xml

```diff
@@ -0,0 +1,175 @@
+    <!-- See: https://github.com/Chi-teck/drupal-coder-extension/pull/5#issuecomment-1427065514 -->
+    <exclude name="SlevomatCodingStandard.ControlStructures.RequireTernaryOperator"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.RequireYodaComparison"/>
+    <exclude name="SlevomatCodingStandard.Exceptions.DisallowNonCapturingCatch"/>
+    <!-- Undocumented. Not suitable for Drupal, because functions grows very heavily. E.g. hook_theme(). -->
+    <exclude name="SlevomatCodingStandard.Files.FunctionLength.FunctionLength"/>
+    <exclude name="SlevomatCodingStandard.Files.FileLength"/>
+    <exclude name="SlevomatCodingStandard.Files.LineLength"/>
+    <exclude name="SlevomatCodingStandard.Files.TypeNameMatchesFileName"/>
+    <exclude name="SlevomatCodingStandard.Functions.DisallowArrowFunction"/>
+    <exclude name="SlevomatCodingStandard.Functions.DisallowNamedArguments"/>
+    <exclude name="SlevomatCodingStandard.Functions.DisallowTrailingCommaInCall"/>
+    <exclude name="SlevomatCodingStandard.Functions.DisallowTrailingCommaInClosureUse"/>
+    <exclude name="SlevomatCodingStandard.Functions.DisallowTrailingCommaInDeclaration"/>
+    <!-- Excluded, because it forces control structures to be multiline in some cases, but Drupal discourages that. -->
+    <exclude name="SlevomatCodingStandard.Functions.RequireMultiLineCall"/>
+    <exclude name="SlevomatCodingStandard.Functions.FunctionLength"/>
+    <exclude name="SlevomatCodingStandard.Functions.StrictCall"/>
+    <exclude name="SlevomatCodingStandard.Functions.UnusedParameter"/>
+    <exclude name="SlevomatCodingStandard.Functions.UselessParameterDefaultValue"/>
+    <exclude name="SlevomatCodingStandard.Namespaces.AlphabeticallySortedUses"/>
+    <exclude name="SlevomatCodingStandard.Namespaces.FullyQualifiedExceptions"/>
+    <exclude name="SlevomatCodingStandard.Namespaces.NamespaceSpacing"/>
+    <exclude name="SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly"/>
+    <exclude name="SlevomatCodingStandard.Namespaces.UnusedUses"/>
+    <exclude name="SlevomatCodingStandard.Namespaces.UseOnlyWhitelistedNamespaces"/>
+    <exclude name="SlevomatCodingStandard.Namespaces.UseSpacing"/>
+    <exclude name="SlevomatCodingStandard.Numbers.DisallowNumericLiteralSeparator"/>
+    <exclude name="SlevomatCodingStandard.Operators.DisallowEqualOperators"/>
+    <exclude name="SlevomatCodingStandard.Operators.DisallowIncrementAndDecrementOperators"/>
+    <exclude name="SlevomatCodingStandard.Operators.NegationOperatorSpacing"/>
+    <exclude name="SlevomatCodingStandard.Operators.SpreadOperatorSpacing"/>
+    <exclude name="SlevomatCodingStandard.PHP.DisallowDirectMagicInvokeCall"/>
+    <exclude name="SlevomatCodingStandard.PHP.DisallowReference"/>
+    <exclude name="SlevomatCodingStandard.PHP.ForbiddenClasses"/>
+    <exclude name="SlevomatCodingStandard.PHP.ReferenceSpacing"/>
+    <exclude name="SlevomatCodingStandard.PHP.RequireExplicitAssertion"/>
```

Comment:
I found it very useful, actually. Should we enable it by default? It forces to use `\assert()` instead of `/** @var Foo $bar */`.